widgets/card_widget.php

```diff
@@ -366,7 +366,7 @@ protected function register_controls()
 		$this->add_control(
 			'description_color',
 			[
-				'label' => esc_html__('Color', 'wasyo-image-card'),
+				'label' => esc_html__('Color', 'Image-card'),
```

Comment:
The majority of this pull request is renaming the Text Domain? What terrible structure! The more I use wordpress, the more I dislike it.

In summary: lets remove the Text Domain completely.